tox.ini

```diff
@@ -17,7 +17,8 @@ deps =
     cover: coveralls
 commands =
     nosetest: nosetests
-    pep8: flake8 --exclude lib,bin --max-complexity=10
+    pep8: flake8 --exclude lib,bin --max-complexity=10 \
+    pep8: --import-order-style=google
```

Comment:
my mistake about the import ordering. feel free to use whichever you think most closely matches the rest of the world's style.